include/openthread/platform/crypto.h

```diff
@@ -174,6 +177,19 @@ typedef struct otPlatCryptoEcdsaKeyPair
     uint8_t mDerLength;
 } otPlatCryptoEcdsaKeyPair;
 
+/**
+ * @struct otPlatCryptoEcdsaKeyRef
+ *
+ * This structure represents an ECDSA key pair as a PSA keyRef.
+ *
+ * This structure is only used by OT core when `OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE` is enabled.
+ *
+ */
+typedef struct otPlatCryptoEcdsaKeyRef
+{
+    uint32_t mKeyRef;
+} otPlatCryptoEcdsaKeyRef;
```

Comment:
Do we need `struct`?  We can use:
```
typedef otCryptoKeyRef  otPlatCryptoEcdsaKeyRef
```

We already do the same for Network Key:
```
typedef otCryptoKeyRef otNetworkKeyRef; ///< Reference to Key
```

Reply:
Yes, we can do that. I wanted to keep the keypairref as close to keypair as possible, and that is why I made it a struct.